### DISCUSSION TITLE
Qual processador vc escolheria

### DISCUSSION THREAD

**Discussion by reinaldofox** [frontendbr/forum#2473] [Perguntas e Respostas]
Estou na duvida entre dois notebooks Asus Vivobook, a diferença entre eles é apenas o processador.
Um com Intel Core i5 12450H  e outro com Ryzen 7 4800H.
Qual dos dois seria a melhor opção.
Obs.: Uso apenas para trabalhar como desenvolvedor Fullstack, estudar, assistir filmes e videos, não jogo e nem uso programas pesados, como editores de vídeo e imagens.

**Comment by Diego-Girao** [comment]
<h3>Olá Reinaldo, espero que esteja bem!</h3>

<h4>Eu iria de Ryzen 7 4800H + S.O Linux(A gosto do usuário), digo isso por ter tido uma experiencia melhor com essa configuração, porém era um Samsung Galaxy Book com 16gb ram,  com é uma diferença mínima, será mais um gosto pessoal seu e o valor a ser investido, caso tenha uma grande diferença de valores.</h4>

**Comment by Frankiando** [comment]
Custo benefício eu ficaria com o processador da AMD 5600 G ele é barato é muito bom para trabalhar e bom para jogar. Futuramente vc pode melhorar colocando um placa de vídeo

**Reply by maxxcleiton** [reply]
Ele está se referindo a notebooks, não PCs de mesa.

**Comment by maxxcleiton** [comment]
Eu escolheria o Ryzen, pois o que já ouvi muito de Intel em notebooks é que esquentam desnecessariamente demais.
E que os Ryzen são mais equilibrados em relação a temperatura.

**Comment by miguellS-cr** [comment]
Olá! 
Qual [notebook melhor](https://pt.globy.com/b2bmarket/listing/consumer-electronics/laptops-computers-accessories) para desenvolvimento Front-end: ASUS Vivobook 15 ou Lenovo ThinkBook 16 G7 ARP?

**Comment by hebcodee** [comment]
Eu escolheria o Ryzen
